packages/core/lib/commands/help/meta.js

```diff
@@ -1,9 +1,16 @@
+const OS = require("os");
+
 module.exports = {
   command: "help",
   description:
     "List all commands or provide information about a specific command",
   help: {
-    usage: "truffle help [<command>]",
+    usage:
+      "truffle help [<command> <subCommand>]" +
+      OS.EOL +
+      "                truffle --help [<command> <subCommand>]" +
```

Comment:
Maybe we should remove the descriptions involving subcommands because I don't think that is currently working.

Reply:
Do you mean the help info for the sub commands? Currently `truffle help db serve` does work.

Reply:
Sub-command help works in this branch, and an error message is displayed if the sub-command is invalid.

Reply:
sub-command help also working in stable truffle tho.